nxdialog/man/nxdialog.1

```diff
@@ -0,0 +1,72 @@
+'\" -*- coding: utf-8 -*-
+.if \n(.g .ds T< \\FC
+.if \n(.g .ds T> \\F[\n[.fam]]
+.de URL
+\\$2 \(la\\$1\(ra\\$3
+..
+.if \n(.g .mso www.tmac
+.TH nxdialog 1 "Jan 2019" "Version 3.5.99.18" "NX Dialog"
```

Comment:
Hmm, that date looks strange..

Reply:
> Hmm, that date looks strange..

date bump comes with release, the date is as in all the other man pages